ckanext/dcat/plugins.py

```diff
@@ -26,11 +26,17 @@ class DCATPlugin(p.SingletonPlugin, DefaultTranslation):
     p.implements(p.IConfigurer, inherit=True)
     p.implements(p.IRoutes, inherit=True)
     p.implements(p.IActions, inherit=True)
+    p.implements(p.IFacets, inherit=True)
     p.implements(p.IAuthFunctions, inherit=True)
     p.implements(p.IPackageController, inherit=True)
     if p.toolkit.check_ckan_version(min_version='2.5.0'):
         p.implements(p.ITranslation, inherit=True)
 
+    # IFacets
```

Comment:
I would prefer if this was handled by each CKAN individual instance if they want to, rather than by ckanxet-dcat